src/test/java/org/opentripplanner/routing/graph/GraphSerializationTest.java

```diff
@@ -129,7 +129,9 @@ private static void assertNoDifferences(Graph g1, Graph g2) {
       "tripPatternForId",
       "transitLayer",
       "realtimeTransitLayer",
-      "dateTime"
+      "dateTime",
+      "notesForEdge",
```

Comment:
Shouldn't these equal?

Reply:
`Edge`s don't have an `equals()` method, and so the `Map`s are not equal.